src/annotator/adder.js

```diff
@@ -198,22 +217,26 @@ export class Adder {
       switch (command) {
         case 'annotate':
           this._onAnnotate();
+          this.hide();
           break;
         case 'highlight':
           this._onHighlight();
+          this.hide();
+          break;
+        case 'show':
+          this._onShowAnnotations(this.annotationsForSelection);
```

Comment:
Because the `show` option doesn't subsequently `hide` like the other two commands, it leaves a visible focus ring on the adder: 

<img width="358" alt="Screen Shot 2020-04-03 at 9 02 03 AM" src="https://user-images.githubusercontent.com/439947/78363644-1cf55c00-758a-11ea-94d3-39372403a326.png">

Is this intentional?

Reply:
No. I'll add the same mixin that we use to avoid unwanted focus rings after mouse (not keyboard) input on other buttons.